components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -41,16 +41,16 @@
 
     <!-- The text shown inside the advanced button used to expand the advanced options. It's only shown when a website has an invalid SSL certificate. -->
     <string name="mozac_browser_errorpages_security_bad_cert_advanced">Advanced…</string>
-    <!-- The advanced certificate information shown when a website sends has an invalid SSL certificate. The %1$s will be replaced by the website URL. It's only shown when a website has an invalid SSL certificate. -->
+    <!-- The advanced certificate information shown when a website sends has an invalid SSL certificate. The %1$s will be replaced by the app name and %2$s will be replaced by website URL. It's only shown when a website has an invalid SSL certificate. -->
     <string name="mozac_browser_errorpages_security_bad_cert_techInfo"><![CDATA[
         <label>Someone could be trying to impersonate the site and you should not continue.</label>
         <br><br>
-        <label>Websites prove their identity via certificates. Firefox does not trust <b>%1$s</b> because its certificate issuer is unknown, the certificate is self-signed, or the server is not sending the correct intermediate certificates.</label>
+        <label>Websites prove their identity via certificates. %1$s does not trust <b>%2$s</b> because its certificate issuer is unknown, the certificate is self-signed, or the server is not sending the correct intermediate certificates.</label>
     ]]></string>
```

Comment:
I assume this will need a new string name. @Delphine @Pike Is that correct?

Reply:
Thanks for checking. The string-IDs need to change when strings are already exposed to l10n. As we're holding this one in quarantine in https://github.com/mozilla-l10n/android-l10n/pull/150, this doesn't need an ID change.

Reply:
Ah, great. Thanks!